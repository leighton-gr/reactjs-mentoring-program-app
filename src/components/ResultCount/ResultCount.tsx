import React from 'react';

export type Props = {
    count: number;
}

export const ResultCount = ({ count }: Props) => {
    return (
        <span>{count} movies found</span>
    )
};