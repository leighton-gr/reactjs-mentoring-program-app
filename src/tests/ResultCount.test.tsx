import renderer from 'react-test-renderer';
import { ResultCount } from '../components/ResultCount';
import React from 'react';
import { expect } from '@jest/globals';

describe('Result count test', () => {
    it('renders ResultCount component', () => {
        const tree = renderer.create(<ResultCount count={1}/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});