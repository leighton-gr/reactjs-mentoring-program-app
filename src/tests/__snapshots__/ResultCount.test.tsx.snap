// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result count test renders ResultCount component 1`] = `
<span>
  1
   movies found
</span>
`;
